pkgs/tools/networking/curl/default.nix

```diff
@@ -7,7 +7,7 @@
 , gnutlsSupport ? false, gnutls ? null
 , wolfsslSupport ? false, wolfssl ? null
 , scpSupport ? zlibSupport && !stdenv.isSunOS && !stdenv.isCygwin, libssh2 ? null
-, gssSupport ? with stdenv.hostPlatform; !(
+, gssSupport ? with stdenv.hostPlatform; (
```

Comment:
I'm not sure about this change. Why invert it?

Reply:
Before the change:
gss support is enabled when: on windows, or building a static library or cross compiling to darwin. Given the issues linked in comment, this looks backward to me.